lib/imports.js

```diff
@@ -1,13 +1,65 @@
 import {
   createImportDeclaration,
   createNamedImportSpecifier,
-} from './createNode';
+} from "./createNode";
+
+const helpers = {
+  isExclusivelyDefaultImport: (importNode) =>
+    importNode.specifiers.length === 1 &&
+    importNode.specifiers[0].type === "ImportDefaultSpecifier",
+
+  /**
+   * Find an import specifier (import name) matching the moduleName in question.
+   * If a falsy moduleName argument is passed to the parent fn, this test will
+   * be ignored and will return true.
+   * @param   {Node}    importNode
+   * @returns {Boolean}
+   */
+  isMatchingName: (importNode, moduleName) => {
```

Comment:
now that it's extracted, we should rename it to something like `isImportIncludingSpecifier`.